### PR TITLE
integration And Contract testing impl

### PR DESCRIPTION
**Pull request by hmcts-shajahan**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -306,11 +306,33 @@ dependencies {
   implementation group: 'com.azure', name: 'azure-core', version: '1.21.0'
   implementation group: 'com.azure', name: 'azure-messaging-servicebus', version: '7.4.2'
   implementation group: 'com.microsoft.azure', name: 'azure-servicebus', version: '3.6.5'
+  implementation group: 'au.com.dius.pact.consumer', name: 'junit5', version: '4.1.11'
+  implementation group: 'au.com.dius.pact.consumer', name: 'java8', version: '4.1.11'
   // https://mvnrepository.com/artifact/com.googlecode.json-simple/json-simple
```
Comment (hmcts-kodatala) [inline build.gradle:311]: remove commented line here.

File: build.gradle [hunk 2]
```diff
@@ -306,11 +306,33 @@ dependencies {
...
+  contractTestRuntime("org.junit.jupiter:junit-jupiter-engine:5.8.2")
+  testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:5.8.2"
+  testCompileOnly 'org.apiguardian:apiguardian-api:1.0.0'
+  contractTestImplementation('org.junit.jupiter:junit-jupiter-api:5.8.2')
+
+  contractTestCompile sourceSets.main.runtimeClasspath
+  contractTestCompile sourceSets.test.runtimeClasspath
+
+  testImplementation group: 'net.serenity-bdd', name: 'serenity-core', version: '2.3.10'
+  //testImplementation group: 'net.serenity-bdd', name: 'serenity-cucumber', version: '1.9.50'
```
Comment (hmcts-kodatala) [inline build.gradle:329]: remove commented line

**Review by hmcts-kodatala — COMMENTED** [review]
please remove the commented lines in build tool file

File: build.gradle (hunk 1 above)
Comment (hmcts-shajahan) [reply build.gradle]: removed

File: build.gradle (hunk 2 above)
Comment (hmcts-shajahan) [reply build.gradle]: removed

**Review by hmcts-kodatala — APPROVED** [review]
Ok

**Pull request merged**